src/styles/style.css

```diff
@@ -0,0 +1,285 @@
+*, *::after, *::before{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.nav{
+    display: flex;
+    /* position: fixed; */
+    height: 64px;
+    width: 100%;
+    background-color: #E6EBFE;
+    box-shadow: 2px 2px 2px  #4a54b1;
+    justify-content: space-between;
+    transition: 0.5s;
+    z-index: 1;
+}
+.logo{
+    font-size: 26px;
+    line-height: 64px;
+    padding-left: 18px;
+    background-image: linear-gradient(to right,#152B79 25%, #3250bd);
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    background-clip: text;
+    color: transparent;
+}
```

Comment:
It's too abstract way to make a text with two colors. Either use `.png` or `.svg` file or color the text inside `span` element.